tck/features/ReturnAcceptance.feature

```diff
@@ -98,17 +189,28 @@ Feature: ReturnAcceptanceTest
 
   Scenario: Support column renaming
     Given an empty graph
-      And having executed: CREATE (:Singleton)
-    When executing query: MATCH (a) WHERE id(a) = 0 RETURN a AS ColumnName
+    And having executed: CREATE (:Singleton)
+    When executing query:
+      """
+      MATCH (a)
+      WHERE id(a) = 0
+      RETURN a AS ColumnName
```

Comment:
Yes certainly, but in the context of the TCK we are the authors, and I think that our column names should follow standard convention for variables. Mostly so because of consistency and lack of a better general rule to follow.